Makefile

```diff
@@ -22,6 +25,9 @@ webui/.version: force
 $(addsuffix .lock, $(ALL_POETRY_PROJECTS)): %.lock: force
 	cd $*; poetry lock --no-update;
 
+$(addsuffix .ruff, $(ALL_POETRY_PROJECTS)): %.ruff: force
+	cd $*; poetry run ruff --fix;
```

Comment:
does this work? I thought it was poetry ruff check --fix

Reply:
It does still work.  Let me change it though. This method is deprecated